lnwallet/reservation.go

```diff
@@ -163,7 +163,7 @@ func NewChannelReservation(capacity, fundingAmt btcutil.Amount,
 		// TODO(roasbeef): need to rework fee structure in general and
 		// also when we "unlock" dual funder within the daemon
 
-		if capacity == fundingAmt {
+		if capacity == localFundingAmt {
```

Comment:
Unrelated to this diff (https://github.com/lightningnetwork/lnd/pull/3165/commits/949783387a3b8f170e362c1edef45a45bb1578e6) but noticed while reviewing the whole method: there's a dust check below and above for 0 when it should really be `2 * DefaultDustLimit()`?

Reply:
I think that is covered by the dustcheck online 198?

Reply:
That check doesn't cover the remote case. It does cover the initiator case though, so one of those checks is redundant.

Reply:
You're right, the below check covers the case were we are the initiator, but not when the remote gets a dust balance. I think it is okay for now though, since the remote is the one that should reject it because of the dust (it doesn't hurt us).